CHANGELOG

```diff
@@ -315,6 +318,8 @@ v3.7.3 (2020-10-14)
 * Fixed issue BTS-169: cost estimation for LIMIT nodes showed wrong number of
   estimated items.
 
+* Added compile option USE_JEMALLOC_PROF to enable memory profiling
```

Comment:
This adds the new option to the CHANGELOG of the already-released version 3.7.3.
I guess the CHANGELOG entry should go into the section for version 3.7.5 instead.

Reply:
Moved to 3.7.3.1 section instead.